### ISSUE TITLE
Worker does not work with a blob url

### ISSUE THREAD

**Issue by priyajeet** [mholt/PapaParse#273]
Due to strict same origin restrictions for web workers the workaround to get it working is to make a GET request to the JS file and parse the response as a blob url.

``` javascript
fetch('http://some.cross.domain/path/to/papaparse.js')
.then((response) => response.blob())
.then((papaWorkerBlob) => {
      Papa.SCRIPT_PATH = URL.createObjectURL(papaWorkerBlob);
      Papa.parse(csvUrl, {
            worker: true,
            ...
      }
});
```

This can then be supplied to Papa.SCRIPT_PATH. However papaparse.js expects a query param `?papaworker` for the webworker and this does not seem to work well with blob urls.

Is there any way this can work? (other than duplicating the file and forcing one to be the worker)

**Comment by priyajeet** [comment]
Another way I could get it working by changing the following line to not rely on `?papaparse`

```javascript
IS_PAPA_WORKER = IS_WORKER && typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope
````

And then obviously not appending `?papaparse`

**Comment by pokoli** [comment]
Could you please provide a PR? Thanks in advance.

**Comment by pokoli** [comment]
Fixed by #599
